### PR TITLE
[tui] Aggregate time in detail view

### LINKED ISSUE #28: Aggregation missing in single entry subwindow
When I click on an entry a subwindow opens with entry details. Aggregated entries are not solved properly here.

bs-watch-3

----

Buchungsstreber `2.4.0`
Ubuntu `18.04.5`
Ruby `ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-linux-gnu]` with
* curses `1.3.2`
* rb-inotify `0.10.1`

### PR DESCRIPTION
**Pull request by dgrammlich**
* Fixes #28

### CODE REVIEW

File: lib/buchungsstreber/cli/tui.rb [hunk 1]
```diff
@@ -101,6 +101,7 @@ def redraw
 
           @win.setpos(@win.cury, 7)
           @win.attron(Curses::A_BOLD) { @win.addstr("%sh" % e[:time]) }
+          #$stderr.puts [e].inspect
```
Comment (BuJo) [inline lib/buchungsstreber/cli/tui.rb:104]: Someday we should introduce a real logging system....

**Comment by BuJo** [comment]
Yes, very reasonable.  The way Information on 'what are the currently focused time entries' is shared.. maybe that can be improved.
My thoughts were around real business objects for entries and days as opposed to flat Hashes, I had a few attempts modeling that, but hadn't arrived at a pleasing result yet.

File: lib/buchungsstreber/cli/tui.rb (hunk 1 above)
Comment (dgrammlich) [reply lib/buchungsstreber/cli/tui.rb:104]: Damnit. >.< This wasn't meant to be committed... I really should configure the debugger.

Comment (BuJo) [reply lib/buchungsstreber/cli/tui.rb:104]: Don't beat yourself up, if you look elsewhere in the code, there are sprinkles of debug statements still left all over. ^_^

**Pull request merged**